components/button/button.component.ts

```diff
@@ -141,10 +141,10 @@ export class NzButtonComponent implements OnDestroy, OnChanges, AfterViewInit, A
       // The compiler generates the `ɵɵlistener` instruction which wraps the actual listener internally into the
       // function, which runs `markDirty()` before running the actual listener (the decorated class method).
       // Since we're preventing the default behavior and stopping event propagation this doesn't require Angular to run the change detection.
-      fromEvent<MouseEvent>(this.elementRef.nativeElement, 'click')
+      fromEvent<MouseEvent>(this.elementRef.nativeElement, 'click', { capture: true })
         .pipe(takeUntil(this.destroy$))
         .subscribe(event => {
-          if (this.disabled && (event.target as HTMLElement)?.tagName === 'A') {
+          if ((this.disabled && (event.target as HTMLElement)?.tagName === 'A') || this.nzLoading) {
```

Comment:
Make sure [this demo](https://ng.ant.design/components/button/zh#components-button-demo-loading) work well, when we change `nzLoading` manually, it will not work as we except.
```typescript
<button nz-button nzType="primary" (click)="loadOne()" [nzLoading]="isLoadingOne">Click me!</button>

loadOne(): void {
  this.isLoadingOne = true;
  setTimeout(() => {
    this.isLoadingOne = false;
  }, 5000);
}

```

Reply:
halo, I have checked the preview and it works the same with the prod site? 
Or can you provide more info?

Reply:
It works well now, I think it's caused by my network :(

Reply:
ok ~